dev/com.ibm.ws.microprofile.metrics.common/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -59,6 +59,11 @@ internal.error.CWMMC0006E=CWMMC0006E: An internal error occurred.
 internal.error.CWMMC0006E.explanation=The application server experienced an internal error.
 internal.error.CWMMC0006E.useraction=Gather a set of traces and open a new issue against your application server.
 
+#User configured mpMetrics to use user-defined library for Micrometer binaries without Prometheus Registry JAR
+noPrometheusRegistry.info.CWMMC0008I=CWMMC0008I: The /metrics endpoint is not available as there is no Prometheus Registry available.
```

Comment:
```suggestion
noPrometheusRegistry.info.CWMMC0008I=CWMMC0008I: The /metrics endpoint is not available because no Prometheus Registry is available.
```